app/views/cases/show.html.slim

```diff
@@ -5,9 +5,10 @@
   .alert-green
     = t('alerts.assignment_accepted_html', case_list_url: cases_path)
 
-- if @case.has_attribute?(:escalation_deadline) && @case.within_escalation_deadline?
-  .alert-orange
-    = t('alerts.escalation_warning_html', escalation_deadline: @case.escalation_deadline)
+- unless @case.is_offender_sar?
+  - if @case.has_attribute?(:escalation_deadline) && @case.within_escalation_deadline?
+    .alert-orange
```

Comment:
You can change the current `if` statement here so that the code reads:

```
- if !@case.is_offender_sar? && @case.has_attribute?(:escalation_deadline) && @case.within_escalation_deadline?
```

This line can then be moved into `cases_helper.rb` as function, e.g.

```
def show_escalation_deadline?(kase)
  !kase.is_offender_sar? && 
    kase.has_attribute?(:escalation_deadline) && 
    kase.within_escalation_deadline?
end
```

And then in show.html.slim

```
- if show_escalation_deadline?(@case)
    .alert-orange
      # Existing code here
```

Then you can add a test in cases_helper_spec as well!

Reply:
Applied this, and it works!